app/src/main/java/com/skyyo/samples/features/textSpans/TextSpansScreen.kt

```diff
@@ -0,0 +1,57 @@
+package com.skyyo.samples.features.textSpans
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Button
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.CornerRadius
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.*
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.skyyo.samples.features.textSpans.composables.*
+
+private const val CORNER_RADIUS_VALUE = 20f
+
+@OptIn(ExperimentalTextApi::class)
+@Composable
+fun TextSpansScreen() {
```

Comment:
let's please source as a comment on top of this class:
`//done using https://saket.me/compose-custom-text-spans/`